devices/rtx/gpu/gpu_objects.h

```diff
@@ -227,6 +228,14 @@ struct Image3DData
   vec3 invSize;
 };
 
+struct CompressedImage2DData
```

Comment:
It looks like this structure is 100% identical to the normal Image2D GPU structure -- I think it would make sense to combine them, and perhaps keep the enum so we can differentiate them if needed but keep the same code paths in `evalMaterial.h`?

Reply:
That's a fair point I did miss when rebasing and simplifying the code.
Given what you mentioned, I'd go for completely removing the identical state and the matching enum. Would this work for you?
Both can be reintroduce later on if that's needed.

Reply:
Comment addressed!